packages/algob/src/lib/account.ts

```diff
@@ -72,6 +72,16 @@ export function mkAccountIndex (accountList: Account[]): Accounts {
   return out;
 }
 
+// returns multisignature account address
+export function getMultisigAddress (version: number, threshold: number, accountList: Account[]): any {
```

Comment:
as noted above ,let's rename this function.

Reply:
done